web/src/emoji_picker.ts

```diff
@@ -354,9 +410,11 @@ function maybe_change_focused_emoji($emoji_map, next_section, next_index, preser
             $next_emoji.trigger("focus");
         } else {
             const start = scroll_util.get_scroll_element($emoji_map).scrollTop();
-            $next_emoji.trigger("focus");
-            if (scroll_util.get_scroll_element($emoji_map).scrollTop() !== start) {
-                scroll_util.get_scroll_element($emoji_map).scrollTop(start);
+            if (start) {
+                $next_emoji.trigger("focus");
+                if (scroll_util.get_scroll_element($emoji_map).scrollTop() !== start) {
+                    scroll_util.get_scroll_element($emoji_map).scrollTop(start);
+                }
```

Comment:
This introduces a bug if `start` is 0. Never use falsey checks for something that might be a number; this should just use `!` to assert `scrollTop` was not undefined.